Add srcset info to images chapter (#1238)
plone/documentation: docs/classic-ui/images.md

The `ImageScaling` view explicitly checks the permissions of the current user.
To access image scales, which are normally not accessible to the current user, override the `validate_access` method in `plone.namedfile.scaling.ImageScale`.

> ## srcset configuration

Reviewer: ```suggestion
## `srcset` configuration
```

> In `/@@imaging-controlpanel` Plone allows you to define srcset's.

Reviewer: ```suggestion
In `/@@imaging-controlpanel` Plone allows you to define HTML `srcset` attributes.
```

> A srcset can help the Browser to serve the best fitting image for the current users situation.

Reviewer: ```suggestion
A `srcset` can help the browser serve the best fitting image for the current users situation.
```

Which image scale is best for the user can be decided on different metrics.

### viewport size

The first is metric would be the viewport size.

```json
{
  "scale": "large",
  "media": "(max-width:800px)"
},
{
  "scale": "larger"
}
```

> With this definition the Browser will use the large scale (800px) when the viewport size is max 800px.

Reviewer: ```suggestion
With this definition the browser will use the `large` scale when the viewport's width is 800 pixels or fewer.
```

> When the viewport size is greater then 800px, the Browser will use the larger scale (1000px).

Reviewer: ```suggestion
When the viewport width is greater than 800 pixels, the browser will use the larger scale `1000px`.
```

### Pixel density

> Another metric is the pixel density of the users screen.

Reviewer: ```suggestion
Another metric is the pixel density of the user's screen.
```

> This metric defines how many DPI per pixel the Screen is using.

Reviewer: ```suggestion
This metric denotes the pixel density, or resolution, of an output device in dots per inch (DPI).
```

```json
{
  "scale": "huge",
  "media": "(min-resolution: 192dpi), (-webkit-min-device-pixel-ratio: 2)"
},
{
  "scale": "large"
}
```

> With this definition the Browser will use the huge scale (1600px) when the screen has a density of 192 DPI per pixel also knows as `2x`.

Reviewer: ```suggestion
With this definition the browser will use the `huge` scale of 1600 pixels when the screen has a density of 192 DPI, also knows as `2x`.
```

We use here two different media queries to also support older Safari versions.

> Mobile devices with Safari like iPhones are still only supporting the old non-standard media query.

Reviewer: ```suggestion
Mobile devices with Safari-like iPhones only support the old non-standard media query.
```

> If you don't care about IE support you can also use `min-resolution: 2dppx` which is closer to `2x`.

Reviewer: ```suggestion
If you do not care about IE support, you can use `min-resolution: 2dppx`, which is closer to `2x`.
```

The most common variants are:

> - `1x`: 96 DPI per pixel

Reviewer: Remove ` per pixel` after DPI for this and all other instances below.

```suggestion
- `1x`: 96 DPI
```
